sdk/avalon_sdk/work_order/work_order_params.py

```diff
@@ -41,17 +42,17 @@ def __init__(
         self.set_worker_id(worker_id)
         self.set_workload_id(workload_id)
         self.set_requester_id(requester_id)
-        if worker_encryption_key:
+        if worker_encryption_key and worker_encryption_key.strip():
             self.set_worker_encryption_key(
                 worker_encryption_key.encode("UTF-8").hex())
         if data_encryption_algorithm:
             self.set_data_encryption_algorithm(data_encryption_algorithm)
-
-        encrypted_session_key = crypto_utility.generate_encrypted_key(
-            session_key, worker_encryption_key)
-        self.set_encrypted_session_key(
-            crypto_utility.byte_array_to_hex(encrypted_session_key)
-        )
+        if worker_encryption_key and worker_encryption_key.strip() and \
```

Comment:
Why do we need `worker_encryption_key.strip()` and `session_key.strip()` ? Ideally there should not be any leading/trailing spaces in worker_encryption_key or session_key.  If it's there then it's an error case

Reply:
yes, blank spaces when passed in worker_encryption_key is not considered as None and hence  the code breaks at crypto utility function. Its a negative case which needs to be filtered during parameter check itself.

Reply:
We should not filter it. If blank spaces are passed either check if it's a valid hex string or let the crypto fail and return crypto error.

Reply:
We should not filter it and remove spaces. If blank spaces or any invalid characters are passed are passed either check if it's a valid hex string or let the crypto fail and return crypto error.

Reply:
strip function is removed